chia/rpc/wallet_rpc_api.py

```diff
@@ -2261,7 +2271,7 @@ async def nft_set_did_bulk(self, request):
         for tx in tx_list:
             if tx.spend_bundle is not None:
                 spend_bundles.append(tx.spend_bundle)
-                refined_tx_list.append(dataclasses.replace(tx, spend_bundle=None))
+            refined_tx_list.append(dataclasses.replace(tx, spend_bundle=None))
```

Comment:
is this deliberate? If so, what does it fix?
(same question applies to the other similar changes)

Reply:
It's gone now.